src/Cake.Core/Extensions/CakeConfigurationExtensions.cs

```diff
@@ -43,5 +43,23 @@ public static DirectoryPath GetModulePath(this ICakeConfiguration configuration,
             var toolPath = configuration.GetToolPath(defaultRoot, environment);
             return toolPath.Combine("Modules").Collapse();
         }
+
+        /// <summary>
+        /// Gets the script cache directory path.
+        /// </summary>
+        /// <param name="configuration">The Cake configuration.</param>
+        /// <param name="defaultRoot">The default root path.</param>
+        /// <param name="environment">The environment.</param>
+        /// <returns>The script cache directory path.</returns>
+        public static DirectoryPath GetScriptCachePath(this ICakeConfiguration configuration, DirectoryPath defaultRoot, ICakeEnvironment environment)
```

Comment:
Same as with constants. Move this extension to Cake.

Reply:
Moved to Cake/Extensions/CakeConfiguration.cs